project/project.go

```diff
@@ -16,6 +15,8 @@ import (
 	"github.com/docker/libcompose/project/events"
 	"github.com/docker/libcompose/utils"
 	"github.com/docker/libcompose/yaml"
+	"os"
```

Comment:
It's a nit but, these imports should be on top (with other built-in ones). It's weird gofmt didn't fail on that.